example/hello.cpp

```diff
@@ -0,0 +1,31 @@
+
+// Range v3 library
+//
+//  Copyright Jeff Garland 2017
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/ericniebler/range-v3
+// 
+
+///[hello]
+#include <range/v3/all.hpp>  //get everything
+namespace rng = ranges::v3;  //easy access
```

Comment:
This alias is never used.

Reply:
gone in my working copy.  This alias was used extensively in the range_by_example code, but wanted these examples to match all the other docs.